### PR TITLE
Pieces of Ember post - Part 1

### PR DESCRIPTION
**Pull request by hbrysiewicz**
This is part 1 of a 3 part series on leveraging the ember ecosystem outside of the ember ecosystem.

### CODE REVIEW

File: source/posts/2016-08-12-pieces-of-ember-part1.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+title: "Pieces of Ember: Part 1"
+author: "Heather Brysiewicz"
+twitter: "caligoanimus"
+github: "hbrysiewicz"
+published: true
+tags: ember, javascript
+summary: "How to leverage the ember ecosystem without the ember ecosystem"
+---
+
+I've been working with [Ember][ember] for years now, even though it feels like just yesterday. Not everyone has had this opportunity and I am often met with questions from inquisitive developers and engineers about the state of ember.
```
Comment (bcardarella) [inline source/posts/2016-08-12-pieces-of-ember-part1.md]: Do you want to capitalize "ember" at the end of this paragraph?

File: source/posts/2016-08-12-pieces-of-ember-part1.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+They know that Ember is an ambitious framework. They've heard it can help boost their productivity. Yet, they hesitate. The project put in their hands is "basically just a landing page" or they believe most frameworks to be excessive - and this concern can be valid. Ember is a mature framework with opinions, build tools, and blueprints that enable me, an ember developer, to focus on engineering an application rather than boilerplate and redundancy.
+
+What these developers probably don't know is that many of the core pieces that make the Ember ecosystem the beautiful powerhouse that it is are also available as micro-libraries.
+
+<br><br>
+<div style="text-align:center;">
+  <img src="/images/2016-07-10-pieces-of-ember/ember-building-blocks.png">
```
Comment (bcardarella) [inline source/posts/2016-08-12-pieces-of-ember-part1.md]: We tend to host these assets on imgur then refer to the url directly with markdown syntax

**Comment by bcardarella** [comment]
This is great, one change I'd make is if this is going to be a 3-part series you may want to briefly discuss what will be in each of the three posts. This way people will have some context for what is to come.

"In Part 1 we'll discuss A, B, C. In Part 2 well discuss, D, and E. Part 3 will conclude with F, G, H and I"

File: source/posts/2016-08-12-pieces-of-ember-part1.md (hunk 1 above)
Comment (hbrysiewicz) [inline source/posts/2016-08-12-pieces-of-ember-part1.md]: Then I would need to capitalize it everywhere... seems excessive?

**Comment by hbrysiewicz** [comment]
I do speak about the different topics in each on line 45~ @bcardarella

**Comment by rsocci** [comment]
great post @hbrysiewicz!

**Pull request merged**